### PR TITLE
Please Adding CW20 Token From Garuda Universe Ecosystem Terra Classic

### PR DESCRIPTION
**Pull request by garudawork**

### CODE REVIEW

**Comment by garudawork** [comment]
hi @HeesungB maybe you can update node.js so our contact can be
 validate

**Comment by HeesungB** [comment]
@garudawork Hello.
I think it's the case of the token name, can you check?

```
Fetched metadata is not same with the token metadata: (
expected: {"name":"GARUDAX TOKEN","symbol":"GRDX","decimals":6}, 
actual: {"name":"GarudaX Token","symbol":"GRDX","decimals":6}), 
contract: terra12f3f5fzfzxckc0qlv3rmwwkjfhzevpwmx77345n0zuu2678vxf0sm6vvcw, chain: columbus)
```

**Comment by garudawork** [comment]
Thank you for the reply sir i will double check it

**Comment by HeesungB** [comment]
@garudawork Could you add image below link?
https://github.com/chainapsis/keplr-contract-registry/tree/main/images

**Comment by cherifabilqis** [comment]
> @garudawork Could you add image below link? https://github.com/chainapsis/keplr-contract-registry/tree/main/images

Done thank you so much

File: cosmos/columbus/tokens/terra12f3f5fzfzxckc0qlv3rmwwkjfhzevpwmx77345n0zuu2678vxf0sm6vvcw.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "contractAddress": "terra12f3f5fzfzxckc0qlv3rmwwkjfhzevpwmx77345n0zuu2678vxf0sm6vvcw",
+    "imageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-contract-registry/020d747f32130bca706ef79e6dd84e2adf1957f7/images/columbus/GARUDAX%20TOKEN.png",
```
Comment (HeesungB) [inline cosmos/columbus/tokens/terra12f3f5fzfzxckc0qlv3rmwwkjfhzevpwmx77345n0zuu2678vxf0sm6vvcw.json:3]: @garudawork Could you rename image as symbol?

**Comment by garudawork** [comment]
is something still wrong?

File: cosmos/columbus/tokens/terra12f3f5fzfzxckc0qlv3rmwwkjfhzevpwmx77345n0zuu2678vxf0sm6vvcw.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "contractAddress": "terra12f3f5fzfzxckc0qlv3rmwwkjfhzevpwmx77345n0zuu2678vxf0sm6vvcw",
+    "imageUrl": "https://raw.githubusercontent.com/garudawork/keplr-contract-registry/main/images/columbus/GARUDAX.png",
```
Comment (HeesungB) [inline cosmos/columbus/tokens/terra12f3f5fzfzxckc0qlv3rmwwkjfhzevpwmx77345n0zuu2678vxf0sm6vvcw.json:3]: @garudawork Could you change repository name of imageUrl?

`https://raw.githubusercontent.com/garudawork`-> `https://raw.githubusercontent.com/chainapsis`

**Comment by HeesungB** [comment]
@garudawork Could you resize image to 512x512px?
```
Reduce image size to 512x512 or smaller (expected: 512x512, actual: 925x926)
```

**Pull request merged**